<launch>
    <group ns="/flying_turtle">
        <param name="turtlebot_aruco_id" value="971" />
        <param name="target_aruco_id" value="212" />
        <param name="camera_index " value="0" />
        <node ns="/flying_turtle" name="aruco_reader" pkg="flying_turtle" type="aruco_reader.py" output="screen" />
        <node ns="/flying_turtle" name="main" pkg="flying_turtle" type="main.py" output="screen" />
    </group>

    <!-- Turtlebot3 -->
    <include file="$(find turtlebot3_bringup)/launch/turtlebot3_remote.launch">
        <arg name="model" value="burger" />
      </include>

      <!-- move_base -->
      <include file="$(find turtlebot3_navigation)/launch/move_base.launch">
        <arg name="model" value="burger" />
        <arg name="move_forward_only" value='true'/>
      </include>

</launch>
